### PR TITLE
Update detection result details

### PR DESCRIPTION
**Pull request by tmoney-bw**
## For the Committer

⚠️ Ensure that for this repo (**bandwidth.github.io**) that the pull request change is opened against the branch `stop-gap-v2`

## Brief Summary of changes

Add links for machine detection result types
Correct example response

### CODE REVIEW

**Comment by DX-Bandwidth** [comment]
http://bw-gitbook-api-docs-vapi-1-update-amd-docs.s3-website-us-east-1.amazonaws.com/

File: voice/bxml/callbacks/about.md [hunk 1]
```diff
@@ -47,6 +47,7 @@ URLs, so POST is the preferred method.
 | [Conference Recording Available](conferenceRecordingAvailable.md) | Bandwidth API sends this when a recording started by a `<StartRecording>`in a conference is available for download. |
 | [Disconnect](disconnect.md)                    | Bandwidth API sends this when a call ends.                                                                                       |
 | [DTMF](dtmf.md)                                | Bandwidth API sends this for every digit detected after a `<StartGather>` is executed. |
+| [Machine Detection Complete](machineDetectionComplete.md)   | Bandwidth API sends this when the result of a machine detection request is available. |
```
Comment (nashley) [inline voice/bxml/callbacks/about.md:50]: I'd line up the `|`, but you really don't need to

File: voice/bxml/callbacks/answer.md [hunk 2]
```diff
@@ -30,7 +30,7 @@ Content-Type: application/xml; charset=utf-8
 | startTime     | Time the call was started, in ISO 8601 format. |
 | answerTime    | Time the call was answered, in ISO 8601 format. |
 | tag           | (optional) The `tag`  specified on call creation. If no `tag` was specified or it was previously cleared, this field will not be present. |
-| machineDetectionResult | (optional) if a sync machine detection operation was requested, the result of the operation will be sent here. |
+| machineDetectionResult | (optional) if a sync machine detection operation was requested, the result of the operation will be sent here. Possible values are the same as for the async webhook [Machine Detection Complete](machineDetectionComplete.md) |
```
Comment (nashley) [inline voice/bxml/callbacks/answer.md]:
```suggestion
| machineDetectionResult | (optional) if [machine detection was requested](../../methods/calls/postCalls.md#machine-detection-request) in `sync` mode, the result will be specified here. Possible values are the same as the async counterpart: [Machine Detection Complete](machineDetectionComplete.md) |
```
Not sure if my suggested link works, but I think it should.
I still don't like `webhook`.

**Comment by DX-Bandwidth** [comment]
http://bw-gitbook-api-docs-vapi-1-update-amd-docs.s3-website-us-east-1.amazonaws.com/

**Review by nashley — APPROVED** [review]
I confirmed the links work

**Review by nashley — CHANGES_REQUESTED** [review]
Example is incorrect

**Review by ajrice6713 — DISMISSED** [review]

**Comment by DX-Bandwidth** [comment]
http://bw-gitbook-api-docs-vapi-1-update-amd-docs.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
http://bw-gitbook-api-docs-vapi-1-update-amd-docs.s3-website-us-east-1.amazonaws.com/

**Review by nashley — APPROVED** [review]

**Review by nashley — CHANGES_REQUESTED** [review]
Bad json

**Comment by DX-Bandwidth** [comment]
http://bw-gitbook-api-docs-vapi-1-update-amd-docs.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
http://bw-gitbook-api-docs-vapi-1-update-amd-docs.s3-website-us-east-1.amazonaws.com/

**Review by nashley — APPROVED** [review]
Indentation is still inconsistent, but whatever

**Review by vsmith-bw — APPROVED** [review]

**Pull request merged**